From 7cc71c2e2f3cf9f486210132936db718c3c260b6 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Fri, 2 Nov 2018 09:02:31 +0800
Subject: [PATCH] drm/bridge: synopsys: dw-hdmi: set print level to debug if
 not support hdr

If encoder does not support HDR, the DRM_ERROR print in
hdmi_config_hdr_infoframe would be misleading to there is
abnormal error in drm/hdmi. Now change it to DRM_DEBUG.

Change-Id: Icdaa4ea408542c6428e22948522ed2017475ea18
Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/gpu/drm/bridge/synopsys/dw-hdmi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c b/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
index 8cb2cb4e61a6..9ee0a87b45e1 100644
--- a/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
+++ b/drivers/gpu/drm/bridge/synopsys/dw-hdmi.c
@@ -1826,7 +1826,7 @@ static void hdmi_config_hdr_infoframe(struct dw_hdmi *hdmi)
 
 	/* Dynamic Range and Mastering Infoframe is introduced in v2.11a. */
 	if (hdmi->version < 0x211a) {
-		DRM_ERROR("Not support DRM Infoframe\n");
+		DRM_DEBUG("Not support DRM Infoframe\n");
 		return;
 	}
 
-- 
2.35.3

